cmd/ddev/cmd/snapshot.go

```diff
@@ -26,17 +32,59 @@ ddev snapshot --all`,
 		}
 
 		for _, app := range apps {
-			if snapshotNameOutput, err := app.Snapshot(snapshotName); err != nil {
-				util.Failed("Failed to snapshot %s: %v", app.GetName(), err)
-			} else {
-				util.Success("Created snapshot %s", snapshotNameOutput)
+
+			switch {
+			case snapshotList:
+				listAppSnapshots(app)
+			case snapshotCleanup:
+				deleteAppSnapshot(app)
```

Comment:
I wondered if this should have a util.Confirm() before doing it. Should people be asked before deleting all the snapshots?

Reply:
Any deletion requires to add an explicit option. So it cannot happen accidentally.
Maybe we should add the confirmation in combination with the  "--all" option?
If we add this, it would be useful to have a `--force` option as well.

Reply:
There's a global --yes/-y option. We'd just have to check that. 

You say 
> Any deletion requires to add an explicit option

but I'd say that a util.Confirm here would be good, overridden with the default --yes/-y

Reply:
Weird, I'm pretty sure it's supposed to be a global flag, but the best example for you (in delete.go) doesn't do it globally at all.

Reply:
I added a confirmation option. The delete command additionally forbid to combine `--all` and `--yes`. I did not port that logic, because it's useful to cleanup all snapshots. What do you think?